### ISSUE TITLE
Backtesting --export=trades broken

### ISSUE THREAD

**Issue by glonlas** [gcarq/freqtrade#535]
## Step 2: Describe your environment

  * Python Version: Python 3.6.4 (`python -V`)
  * Branch: Master | Develop
  * Last Commit ID: b799445b1a690a3773cb4b0ab73947c382285f95 (`git log --format="%H" -n 1`)
 
## Step 3: Describe the problem:
The backtesting export is broken and throw the error `TypeError: Object of type 'Timestamp' is not JSON serializable`. This is caused by the Timestamp object.

This issue is already fixed in the branch `feature/objectify`.

### Steps to reproduce:

  1. Run a backtesting with the param --export=trades `python3 freqtrade/main.py backtesting --export=trades`

### Relevant code exceptions or logs:

  ```
Traceback (most recent call last):
  File "freqtrade/main.py", line 564, in <module>
    main(sys.argv[1:])
  File "freqtrade/main.py", line 498, in main
    args.func(args)
  File "/freqtrade/freqtrade/optimize/backtesting.py", line 226, in start
    'record': args.export
  File "freqtrade/freqtrade/optimize/backtesting.py", line 154, in backtest
    misc.file_dump_json('backtest-result.json', records)
  File "freqtrade/freqtrade/misc.py", line 65, in file_dump_json
    json.dump(data, fp)
  File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/__init__.py", line 179, in dump
    for chunk in iterable:
  File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 428, in _iterencode
    yield from _iterencode_list(o, _current_indent_level)
  File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 325, in _iterencode_list
    yield from chunks
  File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 325, in _iterencode_list
    yield from chunks
  File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 437, in _iterencode
    o = _default(o)
  File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 180, in default
    o.__class__.__name__)
TypeError: Object of type 'Timestamp' is not JSON serializable
  ```

**Comment by xmatthias** [comment]
not sure if feature/objectify fixes this correctly - it changes the extract-format to date, where versions before c62356438a60ec40c7e724fa4f0e8dff898751f4  exported ticker-id, not date of the trade (which is covered in fields 2 and 3 - but in a different format).

**Comment by gcarq** [comment]
Already fixed in develop @ 9cb55910077b673dd2fa8664b0cb765dae21f250.
